src/js/tabs/pid_tuning.js

```diff
@@ -451,8 +451,20 @@ TABS.pid_tuning.initialize = function (callback) {
             });
             ffInterpolateCheck.change();
 
+            // Vbat Sag Compensation
+            const vbatSagCompensationCheck = $('input[id="vbatSagCompensation"]');
+
+            vbatSagCompensationCheck.prop('checked', ADVANCED_TUNING.vbat_sag_compensation !== 0);
+            $('input[name="vbatSagValue"]').val(ADVANCED_TUNING.vbat_sag_compensation > 0 ? ADVANCED_TUNING.vbat_sag_compensation : 100);
+
+            vbatSagCompensationCheck.change(function() {
+                const checked = $(this).is(':checked');
+                $('.vbatSagCompensation .suboption').toggle(checked);
+            });
+            vbatSagCompensationCheck.change();
```

Comment:
You can add the change action directly to the change function, something like:
```
vbatSagCompensationCheck.change(function() {
                const checked = $(this).is(':checked');
                $('.vbatSagCompensation .suboption').toggle(checked);
            }).change();

Reply:
okey, I dont know why I not included directly in the function since I made FF Interpolate, making changes now